lua/telescope/actions/mt.lua

```diff
@@ -123,28 +140,35 @@ action_mt.create = function(mod)
   return mt
 end
 
-action_mt.transform = function(k, mt, mod, v)
+action_mt.transform = function(k, mt, v)
```

Comment:
this is technically a breaking change, we could just use `_` for now if we wanted. Or just break it if we don't think anyone is really using it.